app/main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100,
+                 hidden: bool = False) -> None:
```

Comment:
One attribute per line, when you have a lot of attributes. At checklist you have a good example of styling
```
    def long_function_name(
            var_one,
            var_two,
            var_three,
            var_four
    ) -> None:
```